### ISSUE TITLE
Add shapes

### ISSUE THREAD

**Issue by ashahabov** [ShapeCrawler/ShapeCrawler#417]
## Non Auto Shape

- [x] `IShapeCollection.AddTable()`
- [x] `IShapeCollection.AddPicture()`
- [x] _`IShapeCollection.AddVideo()`_
- [x] _`IShapeCollection.AddAudio()`_

### Block Arrows
- [x] `IShapeCollection.AddArrowRight()`
- [x] `IShapeCollection.AddArrowLeft()`

### Flowchart
- [x] `IShapeCollection.AddFlowchartProcess()`

**Comment by jcoliz** [comment]
This is largely handled by #807. Also add picture has long been done, right?

**Comment by ashahabov** [comment]
Correct. `IShapeCollection.AddPicture()` is already done. The rest should be tested. Will do this.
